client/server/pages/index.js

```diff
@@ -628,10 +632,12 @@ export default function pages() {
 					);
 				} else if ( ! config.isEnabled( 'jetpack-cloud/connect' ) ) {
 					res.redirect( 'https://wordpress.com/pricing' );
+				} else {
+					next();
 				}
+			} else {
+				next();
 			}
-
-			next();
```

Comment:
This change causes `res.redirect()` to be called and then `next()`, too. Before this patch, `next()` was not called in case of redirect, interrupting the handler chain. Was this an intent?

Reply:
Oh, I just noticed I read the patch backwards 🙂 It's fixing the exact bug I'm describing, not introducing it!